plan/config_test.go

```diff
@@ -116,3 +127,74 @@ func TestParseBehavior(t *testing.T) {
 		assert.Equal(t, tt.want, parseBehavior(tt.give))
 	}
 }
+
+func TestParseSkipBehavior(t *testing.T) {
+	tests := []struct {
+		give       string
+		desc       string
+		wantFilter []Filter
+		wantKey    string
+		wantError  error
+	}{
+		{
+			give: "foo=client:c+server:b",
+			desc: "single filter.",
+			wantFilter: []Filter{
+				{
+					AxisMatches: map[string]string{
+						"client": "c",
+						"server": "b",
+					},
+				},
+			},
+			wantKey:   "foo",
+			wantError: nil,
+		},
+		{
+			give: "x=a:b,c:d",
+			desc: "multiple filters.",
+			wantFilter: []Filter{
+				{
+					AxisMatches: map[string]string{
+						"a": "b",
+					},
+				},
+				{
+					AxisMatches: map[string]string{
+						"c": "d",
+					},
+				},
+			},
+			wantKey:   "x",
+			wantError: nil,
+		},
+		{
+			give: "foo=client:weird:name+server:b",
+			desc: "single filter testing name separation.",
+			wantFilter: []Filter{
+				{
+					AxisMatches: map[string]string{
+						"client": "weird:name",
+						"server": "b",
+					},
+				},
+			},
+			wantKey:   "foo",
+			wantError: nil,
+		},
+		{
+			give:       "x=a:b,,c:d",
+			desc:       "invalid filters",
+			wantFilter: nil,
+			wantKey:    "",
+			wantError:  fmt.Errorf("Invalid filter definition:  in input x=a:b,,c:d"),
+		},
+	}
+
+	for _, tt := range tests {
+		key, filter, err := parseSkipBehavior(tt.give)
```

Comment:
You must check `err`. For example: `if assert.NoError(t, err) { further assertions }`